app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,75 @@
+class PostsController < ApplicationController
+  before_action :user_get
+  before_action :set_post, only: %i[show edit update destroy]
```

Comment:
* [ ] **_Consider the use of `before_action` to restrict access to the `:new` and `:create` methods ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/before_action_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))_**